core/src/main/java/com/linecorp/armeria/common/HttpResponseWriter.java

```diff
@@ -140,7 +156,10 @@ default void respond(HttpStatus status, MediaType mediaType, HttpData content) {
      * @param mediaType the {@link MediaType} of the response content
      * @param content the content of the response
      * @param trailingHeaders the trailing HTTP headers
+     *
+     * @deprecated Use {@link HttpResponse#of(HttpStatus, MediaType, HttpData, HttpHeaders)}.
      */
+    @Deprecated
     default void respond(HttpStatus status, MediaType mediaType, HttpData content,
```

Comment:
Don't we need to deprecate the `response(AggregatedHttpMessage)` method as well?

Reply:
Oops~ I misread forget about this. :-)